docs/porting/display.rst

```diff
@@ -86,7 +86,7 @@ Example:
 .. code:: c
 
    static uint16_t buf[LCD_HOR_RES * LCD_VER_RES / 10];
-   lv_disp_set_draw_buffers(disp, buf, NULL, sizeof(buf), LV_DISP_RENDER_MODE_PARTIAL);
+   lv_display_set_buffers(disp, buf, NULL, sizeof(buf), LV_DISP_RENDER_MODE_PARTIAL);
```

Comment:
Render mode enum new name: `LV_DISPLAY_RENDER_MODE_PARTIAL`

Reply:
There are lots of `disp` `indev` in doc right now that deserves another PR for complete replacement. I tried to update in this PR but found too many and gave up.

Let's focus on the recent `lv_display_set_buffers` for now.

Reply:
OK, then there will be separate PRs to update remaining old names.